sechub-integrationtest/src/test/java/com/mercedesbenz/sechub/integrationtest/scenario22/PDSPrepareIntegrationScenario22IntTest.java

```diff
@@ -225,4 +228,33 @@ private void approveJobAndSimulateSourceCodeAvailable(TestProject project, UUID
           /* @formatter:on */
     }
 
+    @Test
+    public void startPDSPrepareJobFromRemoteCodeScanConfigurationwithoutClient() {
```

Comment:
```suggestion
    public void start_PDS_prepare_job_from_remote_codescan_configuration() {
```


At our tests, we use always lowercased method names with _ separation (easier to read and always clear this is a test method)